src/utils/metamask.ts

```diff
@@ -1,9 +1,16 @@
 // Copyright (c) Silence Laboratories Pte. Ltd.
 // This software is licensed under the Silence Laboratories License Agreement.
 
+const versionParsing = (version: string) => {
```

Comment:
"parseSemverAsArray" could be a better name

Reply:
because it shows that the function is extracting semver from version param and convert to array without needing to look into the function body